kafka-rest/src/test/java/io/confluent/kafkarest/integration/ClusterTestHarness.java

```diff
@@ -424,6 +437,160 @@ protected Properties getBrokerProperties(int i) {
     return props;
   }
 
+  private static boolean shouldEnable(
+      Option<SecurityProtocol> interBrokerSecurityProtocol, SecurityProtocol protocol) {
+    if (interBrokerSecurityProtocol.isDefined()) {
+      return interBrokerSecurityProtocol.get() == protocol;
+    }
+    return false;
+  }
+
+  public static Properties createBrokerConfig(
```

Comment:
@trnguyencflt is this taken from AK? If true can you add a comment linking to the source. That way its easy to track the source of there are any divergences in the future, and need to update.

Reply:
added 👍